src/main/java/org/prebid/server/handler/openrtb2/AmpHandler.java

```diff
@@ -144,6 +147,44 @@ public void handle(RoutingContext routingContext) {
                 .setHandler(responseResult -> handleResult(responseResult, ampEventBuilder, routingContext, startTime));
     }
 
+    private AuctionContext validateAccount(AuctionContext context) {
+        if (adminManager.contains(AdminManager.ADMIN_TIME_KEY)) {
+
+            try {
+                final List<String> accountIds = new ArrayList<>();
```

Comment:
I'm concerned about the logic of this method.
What are you trying to achieve here and what is the actual requirement?